pulpcore/cli/common/openapi.py

```diff
@@ -291,6 +293,11 @@ def call(
             parameters = {}
         else:
             parameters = parameters.copy()
+        if self.api_spec["info"].get("x-pulp-domain-enabled", False):
```

Comment:
I know, this is an early draft, but i'd like to keep any thing Pulp-specific out of the `openapi.py` file. Can this be handled in `context.py`?

Reply:
I put the domains check into `context.py`, but I had to add a new method to `OpenApi`, `param_spec(operation_id, param_type)`, so I could know when to add `domains_path` to the parameters.